assets/javascripts/discourse/initializers/admin-menu-webinar-button.js

```diff
@@ -30,7 +31,7 @@ function initialize(api) {
     },
 
     addWebinar() {
-      showWebinarModal(this.topic);
+      showWebinarModal(this.topic, api, modal);
```

Comment:
One thing I learned recently: it's a bad idea to look up an object and then use it inside `modifyClass()` - modifications are done once, so you hang on to an outdated instance of a given object.

Use injections or getOwner+lookup in the added methods instead